routes/authors.js

```diff
@@ -0,0 +1,114 @@
+let express = require('express');
+let router = express.Router();
+
+let ApiUtils = require('../utils/ApiUtils');
+
+let AuthorModel = require('../models/Author');
+
+//TODO add TokenValidators
+
+router.get('/', function (req, res, next) {
+    let authorId = req.body.id;
+
+    if (authorId) {
+        AuthorModel
+            .findOne({"_id": authorId})
+            .then(author => {
+                return res
+                    .status(200)
+                    .json(author)
+            })
+            .catch(reason => {
+                sendApiError(res, 500, "Couldn't download authors: " + reason.message)
+            });
+    } else {
+        AuthorModel
+            .find()
+            .then(authors => {
+                return res
+                    .status(200)
+                    .json(authors)
+            })
+            .catch(reason => {
+                sendApiError(res, 500, "Couldn't download authors: " + reason.message)
+            });
+    }
+});
+
+router.post('/add', function (req, res, next) {
```

Comment:
Jeszcze to ma być putem, a nie postem (jak ja tego nie zauważyłem to ja nie wiem...)

Reply:
poprawione